src/radiotray-ng/main.cpp

```diff
@@ -230,7 +231,21 @@ bool process_command_line_args(int argc, char* argv[])
 
 int main(int argc, char* argv[])
 {
-	if (!process_command_line_args(argc, argv))
+    std::locale::global(std::locale(""));
+
+#if HAVE_GETTEXT
+    const char * i18n_domain = GETTEXT_DOMAIN;
+
+#ifndef NDEBUG
+    // under debug build search .mo files in cwd
+    bindtextdomain (i18n_domain, ".");
+#endif
+
+    bind_textdomain_codeset(i18n_domain, "UTF-8");
+    textdomain (i18n_domain);
+#endif
+
+    if (!process_command_line_args(argc, argv))
```

Comment:
I'm using tabs instead of spaces.  Please remove them.
I will eventually add a clang-tidy file.

Reply:
you should look at all other changes as well

Reply:
fixed